samples/bluetooth/central_otc/README.rst

```diff
@@ -0,0 +1,23 @@
+.. _central_otc:
+
+Bluetooth: Central OTC
+######################
+
+Overview
+********
+
+Similar to the :ref:`Central <bluetooth_central>` sample, except that this
+application specifically looks for the OTS (Object Transfer) GATT Service,
+and read metadata/write/read to selected ojects.
```

Comment:
I don't understand the ending of that sentence. Did you mean ", and will attempt to read/write to selected objects and read their metadata." ?

Reply:
Instead of only connecting  and disconnecting to a device like <bluetooth_central> does , this sample will read meteada , write data, read data to specified object. In OTS the element we are going to manipulate called selected object.